master/src/main/java/eu/unifiedviews/master/api/PipelineResource.java

```diff
@@ -48,13 +59,24 @@
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
     public PipelineDTO createPipeline(PipelineDTO pipelineDTO) {
+        // try to get user
+        User user =  userFacade.getUserByExtId(pipelineDTO.getUserExternalId());
+        if(user == null) {
+            throw new ApiException(Response.Status.NOT_FOUND, String.format("User '%s' could not be found! Pipeline could not be created.", pipelineDTO.getUserExternalId()));
+        }
+        // try to get organization
+        Organization organization = userFacade.getOrganizationByName(pipelineDTO.getOrganizationExternalId());
+        if(organization == null) {
+            throw new ApiException(Response.Status.NOT_FOUND, String.format("Organization '%s' could not be found! Pipeline could not be created.", pipelineDTO.getOrganizationExternalId()));
+        }
```

Comment:
If I create pipeline using UV GUI, and config.properties has param "ownership.type" is set to "user", then, the created pipeline has NULL organization. Below is table ppl_model after pipeline "testxx" was created: 

![screen shot 2015-04-16 at 09 27 40](https://cloud.githubusercontent.com/assets/3014917/7176025/f0a0f282-e41a-11e4-8021-8d8480b6ec82.png)


So API should work in a  way:

1) if config.properties "ownership.type" is set to "organization", than it should work as you wrote
2) if config.properties "ownership.type" is set to "user", organization is simply set to NULL. API must NOT throw exception.